src/pages/components/tabs-extended.mdx

```diff
@@ -49,4 +51,32 @@ On mobile, Tabs extended switches to an accordion for easier browsing.
 </Column>
 </Row>
 
+## Options
+
+Tabs extended media is a variation that constructs its content using [Content item horizontal](/components/content-item-horizontal) with media to create an organized section optimized for content with media. This is useful especially when reducing page length is desired. Keep in mind some content will be hidden until user interaction, so they should not be critical information. 
```

Comment:
@RichKummer Looking at this now, I think it would be better to include an H3 here. To match the Image page, this would be "Media".